Training/notebooks/2017v2/scaling_validation.ipynb

```diff
@@ -0,0 +1,3609 @@
+{
```

Comment:
I suggest adding the following validation test: fraction of entries outside of the validity range, which is specified in yaml (e.g. [-5, 5]) should be below a given threshold. Variables for which test fails should be reported and we need to consider a case-by-case basis. As a threshold, I propose to use 0.01.

Reply:
@kandrosov Sounds good, maybe I could propose the following way, which is slightly different from what you suggest but would have a benefit of not rerunning the computation once again. Since in the scaling code I loop firstly over files and then over features, what I could do is for each feature array during aggregation store (per file) the values of it's 0.01 and 0.99 percentiles. This would already give us some intuition on the range of the distribution and also the dynamics of its variation from one file to the other. And this also should be rather straight-forward to implement. Although there ofc can be low statistics for some of the features and percentile estimates will be stochastic... Would it make sense to try it out? Otherwise surely I can implement it the way you suggested.